modules/moving-the-gitops-operator-pod-to-infrastructure-nodes.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assembly:
+//
+// * gitops_workloads_infranodes/run-gitops-control-plane-workload-on-infra-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="moving-the-gitops-operator-pod-to-infrastructure-nodes_{context}"]
+= Moving the {gitops-shortname} Operator pod to infrastructure nodes
+
+You can move the {gitops-shortname} Operator pod to the infrastructure nodes.
+
+.Prerequisites
+* {gitops-title} is installed in your cluster.
+
+.Procedure
+
+. Label existing nodes as infrastructure by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node-name> node-role.kubernetes.io/infra=
+----
++
+.Example output
+[source,terminal]
+----
+node/<node-name> labeled
+----
+
+. Edit the {gitops-title} `Subscription` resource to specify where the Operator should run:
++
+[source,terminal]
+----
+$ oc -n openshift-gitops-operator edit subscription openshift-gitops-operator
+----
+
+. Add `nodeSelector` and `tolerations` to the `spec.config` field in the `Subscription` resource:
++
+.Example Subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-gitops-operator
+  namespace: openshift-gitops-operator
+spec:
+  config:
+    nodeSelector:
+      node-role.kubernetes.io/infra: ""
+    tolerations:
+    - key: node-role.kubernetes.io/infra
+      operator: Exists
+      effect: NoSchedule
+----
++
+.Example output
+[source,terminal]
+----
+subscription.operators.coreos.com/openshift-gitops-operator edited
+----
+
```

Comment:
@reginapizza should we also include the second resolution from [this KCS article ](https://access.redhat.com/solutions/7022926) about control plane pods? I am asking because of the note we have at the beginning of the chapter: "Any other Argo CD instances installed to user namespaces are not eligible to run on infrastructure nodes."

Also, I am not sure how it relates to this procedure about moving the Operator pods to the infra nodes. If that second resolution should be included, where would be the place to put it?

Reply:
We got confirmation here: https://redhat-internal.slack.com/archives/C01RQH8KQ87/p1704720770015969?thread_ts=1701705211.769659&cid=C01RQH8KQ87 so the resolution provided is enough for now.